platform/jvm/capture/src/main/kotlin/io/bitdrift/capture/reports/FatalIssueReportingMode.kt

```diff
@@ -0,0 +1,44 @@
+// capture-sdk - bitdrift's client SDK
+// Copyright Bitdrift, Inc. All rights reserved.
+//
+// Use of this source code is governed by a source available license that can be found in the
+// LICENSE file or at:
+// https://polyformproject.org/wp-content/uploads/2020/06/PolyForm-Shield-1.0.0.txt
+
+package io.bitdrift.capture.reports
+
+/**
+ * Represents the fatal issue reporting configuration mode.
+ *
+ * This sealed class distinguishes between a disabled state (no crash reporting)
+ * and explicitly enabled modes that support different levels of crash reporting.
+ */
+sealed class FatalIssueReportingMode {
+    /**
+     * No fatal issue reporting is enabled.
+     * The SDK will not initialize crash or ANR reporting mechanisms.
+     */
+    object Disabled : FatalIssueReportingMode()
+
+    /**
+     * Represents enabled fatal issue reporting modes.
+     */
+    sealed class Enabled : FatalIssueReportingMode() {
+        /**
+         * Enables reporting of:
+         * - JVM crashes (uncaught exceptions)
+         * - ANR (Application Not Responding) events
+         *
+         * Native (NDK) crash reporting is not enabled in this mode.
+         */
+        object JvmAndAnr : Enabled()
```

Comment:
nit/ also we should consider dropping the `And` here and there to make it more readable